package.json

```diff
@@ -39,6 +42,14 @@
     "eslint": "^6.1.0",
     "eslint-config-airbnb": "^17.1.1",
     "eslint-plugin-import": "^2.18.2",
-    "nodemon": "^1.19.1"
+    "nodemon": "^1.19.1",
```

Comment:
Why isn't this arranged alphabetically?

Reply:
I do not understand why your dev dependencies are not well arranged. This is the correct arrangement.
```
    "@babel/cli": "^7.5.5",
    "@babel/core": "^7.5.5",
    "@babel/node": "^7.5.5",
    "@babel/polyfill": "^7.4.4",
    "@babel/preset-env": "^7.5.5",
    "@babel/register": "^7.5.5",
    "chai": "^4.2.0",
    "coveralls": "^3.0.5",
    "eslint": "^6.1.0",
    "eslint-config-airbnb": "^17.1.1",
    "eslint-plugin-import": "^2.18.2",
    "express": "^4.16.3",
    "mocha": "^6.2.0",
    "nodemon": "^1.19.1",
    "nyc": "^14.1.1",
    "sync-dotenv": "^2.2.0"
```
I hope I don't see this sort of error again. Please try to pay more attention to details.